lib/DidJwtStrategy.js

```diff
@@ -0,0 +1,62 @@
+/*!
+ * Copyright (c) 2017 Digital Bazaar, Inc. All rights reserved.
+ */
+const bedrock = require('bedrock');
+const passport = require('passport');
+const util = require('util');
+const validate = require('bedrock-validation').validate;
+const BedrockError = bedrock.util.BedrockError;
+
+module.exports = Strategy;
+
+/**
+ * Creates a new didJwt Strategy for use with passport.
+ */
+function Strategy(options) {
+  const v = validate('authn-did-jwt.constructor', options);
+  if(!v.valid) {
+    throw v.error;
```

Comment:
I agree that the validation library seems like a good fit.  Are there modifications we can make to the error returned by validation or a new API that would bring it in line.

The validation library seems like the right tool to use rather that writing a bunch of validation code for N parameters.